### ISSUE TITLE
Tour step doesn't show but instead shows the other popover attached to the element

### ISSUE THREAD

**Issue by heliodor** [sorich87/bootstrap-tour#331]
I have an element for which I set up (using Javascript) a popover that is shown when the element is clicked. I also use this element as a step in the tour. When I click the 'next' button to jump to this step in the tour, the tour popover doesn't show and instead the other popover appears.

**Comment by fcanache** [comment]
yeah, unfortunately I am having same issue. Any suggestions in the meanwhile?
